assets/dev/scss/frontend/widgets/image-box.scss

```diff
@@ -67,32 +43,38 @@
 		}
 	}
 
-	@media (max-width: $screen-mobile-max) {
-
-		// TEMP code to override the image spacing
-		.elementor-image-box-img {
-			margin-left: auto !important;
-			margin-right: auto !important;
-			margin-bottom: 15px;
-		}
-	}
-
 	.elementor-image-box-img {
 		display: inline-block; // For alignment content
 	}
 
+	&.elementor-element .elementor-widget-container .elementor-image-box-wrapper .elementor-image-box-img {
+		margin-left: var(--image-margin-left, var(--image-spacing, 15px));
+		margin-right: var(--image-margin-right, var(--image-spacing, 15px));
+		margin-bottom: var(--image-margin-bottom, var(--image-spacing, 15px));
+	}
+
 	.elementor-image-box-title {
 
 		a {
 			color: inherit;
 		}
 	}
 
-	.elementor-image-box-wrapper {
-		text-align: center;
-	}
-
 	.elementor-image-box-description {
 		margin: 0;
 	}
+
+	@media (max-width: $screen-mobile-max) {
+
+		&.elementor-position {
+
+			&-left, &-right {
+
+				// BC compatibility code since 3.7.0
+				.elementor-widget-container .elementor-image-box-wrapper .elementor-image-box-img {
+					margin-bottom: 0;
+				}
```

Comment:
Preferably I would use margin-bottom: 15px, becasue that was the original code from line 76.
But that code was overwritten by the following code from the general styling.

.elementor .elementor-widget:not(.elementor-widget-text-editor):not(.elementor-widget-theme-post-content) figure {
    margin: 0;
}

I am not sure which value I should use.

Reply:
@hein-obox Please elaborate on the comment in the CSS.

Reply:
@hein-obox The original code from line 76 has `margin-bottom: 15px;`?

Reply:
@louiswol94 

The margin-bottom of 15px is overwritten by the :not fix for Gutenberg. 

.elementor .elementor-widget:not(.elementor-widget-text-editor):not(.elementor-widget-theme-post-content) figure {
margin: 0;
}

I have followed the 0px, to keep it BC compatible. 

But I am not a 100% sure if we should stick to 0px or should use 15px. 

Aviv stressed the BC aspect.

Reply:
@hein-obox What I mean is the old code from this widget is `15px`. But you changed it to `0` here. Should you not keep the 15px?

Reply:
@louiswol94 

When you inspect, you'll see this code: 
.elementor .elementor-widget:not(.elementor-widget-text-editor):not(.elementor-widget-theme-post-content) figure {
margin: 0;
}

That forcing a margin-bottom: 0; value, because of the CSS specificity. That is the BC version I am dealing with. That is how it works in the current production version.

I am not introducing the 0, I am merely formalising it. 

But preferably I would use the 15px version. 

Let's double check with @avivu: 
- Originally on mobile we had a margin bottom of 15px. 
- Last year this value changed to 0px, because of some Gutenberg styling (this seems like a bug to me, but this is what is currently how the production version works). 
- In my latest code I am using that margin-bottom: 0; value. Meaning: nothing will change on production, which is good from BC perspective. But now I am kind of formalising a bug.

Reply:
@hein-obox I understand. But because our code was on 15px we should just leave it on 15px in my view. The 0px that was added in the global css probably affected this widget by mistake and does not mean we should `change` this widget's code to 0px.

Reply:
@hein-obox This is a tricky one. Because the original widget intended the `15px`. But the Global CSS of `0px` has been added 2 years ago already, but no one noticed it affected this widget also. So now I guess we have to stick to `0px` so that the Screenshotter can pass.

FYI: https://github.com/elementor/elementor/pull/11998

Reply:
@louiswol94 

This is fascinating. 

Aviv gave me the instruction to use the margin-bottom: 15px;

@avivu 

It is up to you and your colleagues to decide about this.

Reply:
@louiswol94 @avivu 
Final verdict from Aviv: we have decided to use 15px. 
Someone will need to overrule the screenshotter error.